opensrp-opd/src/main/java/org/smartregister/opd/utils/OpdLookUpUtils.java

```diff
@@ -89,7 +89,7 @@ private static String lookUpQuery(@NonNull Map<String, String> entityMap, @NonNu
         queryBUilder.SelectInitiateMainTable(tableName,
                 new String[]{OpdConstants.KEY.RELATIONALID, OpdConstants.KEY.OPENSRP_ID, OpdConstants.KEY.FIRST_NAME, OpdConstants.KEY.LAST_NAME,
                         AllConstants.ChildRegistrationFields.GENDER, OpdConstants.KEY.DOB,
-                        OpdConstants.KEY.BASE_ENTITY_ID}
+                        OpdConstants.KEY.BASE_ENTITY_ID, OpdConstants.KEY.NATIONAL_ID}
```

Comment:
Add DB Constants to the `OpdDbConstants` under the correct table name for cleanliness and easy refactor